plugins/woocommerce/src/Internal/Admin/EmailPreview/EmailPreview.php

```diff
@@ -133,7 +179,12 @@ private function get_dummy_order() {
 		$order->set_billing_address( $address );
 		$order->set_shipping_address( $address );
 
-		return $order;
+		/**
+		 * A dummy WC_Order used in email preview.
+		 *
+		 * @since 9.6.0
+		 */
+		return apply_filters( 'woocommerce_email_preview_dummy_order', $order );
```

Comment:
I think we should attach the email type to assist third parties with using this filter.

```suggestion
		return apply_filters( 'woocommerce_email_preview_dummy_order', $order, $this->email_type );
```

Same with the dummy product/address filters.

Reply:
Addressed in 0f28cf7.